lib/stack/branch/compare.js

```diff
@@ -0,0 +1,78 @@
+import { get } from '../../entity'
+
+/**
+ *
+ * @namespace Branch
+ */
+export function Compare (http, data = {}) {
+  this.stackHeaders = data.stackHeaders
+  this.urlPath = `/compare`
```

Comment:
@nadeem-cs check with Anup if need to update this url as per specification in docs. Or will this remain same as we have in dev-16.

Reply:
As per the latest msg by Anup, v3/stack/branch/compare will be the final route. Should I make the changes acc to that or wait until the final routing is done? @uttamukkoji